feathr_project/feathr/api/app/core/error_handling.py

```diff
@@ -0,0 +1,17 @@
+
+import string
+from fastapi import HTTPException
+from pyapacheatlas.core import (AtlasException)
+from functools import wraps
+from app.core.configs import *
+from app.core.error_code import ErrorCode
+from app.core.feathr_api_exception import FeathrApiException
+
+'''
+This file will keep common error handling by our own logic.
+Make sure all exceptions raised here is registered as FeathrApiException.
+'''
+
+def verifyCode(code:string):
+    if code != appServiceKey:
+        raise FeathrApiException(code=ErrorCode.invalid_code, message="You are not allowed to access this resource")  
```

Comment:
please state what happened(getting 404 from backends etc), direction reason(you already have) and potential solutions or what usres should do.

Reply:
Thanks, enriched description